### DISCUSSION TITLE
download  Scopy app?

### DISCUSSION THREAD

**Discussion by lond131421** [fhdm-dev/scoppy#156] [Q&A]
I am unable to use Google Play. Is there any other way to download the latest Scopy app?

**Comment by GunoX9** [comment]
https://apkpure.com/scoppy-oscilloscope/xyz.fhdm.scoppy

APKMIRROR, APKPURE etc. type of sites provide .apk files most of the apps on Play Store. I can't tell if they are trustable or not but i used them before. I didn't use banking/critical apps(info) on those devices.
